README.md

```diff
@@ -1,56 +1,60 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
Hi @Marlyn-Mayienga,

Good job so far! +1
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight tada champagne
Nice PR description and title +1
Nice README file heavy_check_mark

**Required Changes recycle**
The Github workflows and linters are missing.
 Kindly add them to follow the Microverse Pull Request guidelines in the next PR. Linters are important to help you loom out for errors in your code and correct them before submission.

**Optional suggestions**
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Reply:
@Marlyn-Mayienga  it would be better to use the Github feature to require changes I will leave a link to revisit how you can use it [Reviewing proposed changes in a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)